tests/ImageSharp.Tests/Formats/Jpg/JpegDecoderTests.Metadata.cs

```diff
@@ -288,5 +289,35 @@ public void IgnoreMetadata_ControlsWhetherMetadataIsParsed(bool ignoreMetadata)
                         Assert.Equal(72, imageInfo.Metadata.HorizontalResolution);
                         Assert.Equal(72, imageInfo.Metadata.VerticalResolution);
                     });
+
+        [Fact]
+        public void ExifIfdStructure()
+        {
+            byte[] exifBytes;
+            using var memoryStream = new MemoryStream();
+            using (var image = Image.Load(TestFile.GetInputFileFullPath(TestImages.Jpeg.Baseline.Calliphora)))
+            {
+                var exif = new ExifProfile();
+                exif.SetValue(ExifTag.XPAuthor, Encoding.GetEncoding("UCS-2").GetBytes("Dan Petitt"));
+
+                exif.SetValue(ExifTag.XPTitle, Encoding.GetEncoding("UCS-2").GetBytes("A bit of test metadata for image title"));
+                exif.SetValue(ExifTag.UserComment, new EncodedString("A bit of normal comment text", EncodedStringCode.ASCII));
```

Comment:
@JimBobSquarePants This is API example for `UserComment` tag. 
Doc: https://www.awaresystems.be/imaging/tiff/tifftags/privateifd/exif/usercomment.html

Reply:
Yeah I like it. 👍 I'd swap the arguments in the constructor though.